tools/benchmark_tool/openvino/tools/benchmark/main.py

```diff
@@ -258,7 +256,7 @@ def set_throughput_streams():
             start_time = datetime.utcnow()
             compiled_model = benchmark.core.compile_model(args.path_to_model, benchmark.device)
             duration_ms = f"{(datetime.utcnow() - start_time).total_seconds() * 1000:.2f}"
-            logger.info(f"Compile model took {duration_ms} ms")
+            logger.info(f"Load model took {duration_ms} ms")
```

Comment:
`Compile` seems to be right according to OV API 2.0 naming

Reply:
Rolled back to 'compile'!

Reply:
Don't see that it is rollbacked